README.md

```diff
@@ -1,14 +1,12 @@
 # gsps-trylogia
 
-*Tested with Node.js 18 LTS.*
+*Tested with Node.js 18 & 22 LTS.*
```

Comment:
nit: we could probably drop mentioning the outdated 18 anyway

Reply:
I agree, removed it.